_articles/evaluation_queues.md

```diff
@@ -111,55 +111,55 @@ Every submission you make to an Evaluation queue has a unique ID. This ID should
 
 Navigate to a file in Synapse and click on **Tools** in the upper right-hand corner. Select **Submit To Challenge** to pick the challenge for your submission. Follow the provided steps to complete your submission.
 
-<img style="width: 30%;" id="toobig" src="/assets/images/submit_file_to_challenge.png">
+<img style="width: 30%;" id="toobig" src="../assets/images/submit_file_to_challenge.png">
 
 ## View Submissions of an Evaluation Queue
 
-Submissions can be viewed through leaderboard widgets on Wiki pages.
+Submissions can be viewed and shared with users through [Submission Views]({{ site.baseurl }}{% link _articles/views.md %#create-a-submission-view}) creating dynamic leaderboards. Submission annotations can be added to a SubmissionStatus object and are automatically indexed in the view.
 
-### Adding Leaderboard Widget
+### Creating the Submission View
 
-<img style="width: 80%;" src="/assets/images/add_leaderboard_widget.png">
+Navigate to the **Tables** tab and under the **Table Tools** menu in the upper right-hand select **Add Submission View**:
```

Comment:
Is this still an alpha mode feature?

Reply:
Yep, I noticed that! I'll create a JIRA to take it out of alpha mode.